From: Jakub Jirutka <jakub@jirutka.cz>
Date: Thu, 30 Dec 2021 23:36:48 +0100
Subject: [PATCH] Move imv-wayland and imv-x11 to /usr/libexec

These bins are normally not executed directly, but by /usr/bin/imv script.
Avoid polluting PATH.

--- a/files/imv
+++ b/files/imv
@@ -1,6 +1,6 @@
 #!/bin/sh
 if [ -n "${WAYLAND_DISPLAY}" ]; then
-  exec imv-wayland "$@"
+  exec /usr/libexec/imv-wayland "$@"
 else
-  exec imv-x11 "$@"
+  exec /usr/libexec/imv-x11 "$@"
 fi
--- a/meson.build
+++ b/meson.build
@@ -152,7 +152,7 @@
       [get_variable('files_' + ws), files_imv],
       dependencies: [deps_for_imv, get_variable('deps_for_' + ws)],
       install: true,
-      install_dir: get_option('bindir'),
+      install_dir: get_option('libexecdir'),
     )
   endif
 endforeach
